src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The truck has just started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The truck was stopped");

```

Comment:
Similarly, the message in the 'stopWork' method should be more informative. Consider 'Truck has stopped working.' to clearly indicate what machine has stopped.